src/components/ReportActionItem/MoneyRequestPreview.tsx

```diff
@@ -155,20 +155,25 @@ function MoneyRequestPreview({
     const isScanning = hasReceipt && TransactionUtils.isReceiptBeingScanned(transaction);
     const hasFieldErrors = TransactionUtils.hasMissingSmartscanFields(transaction);
     const isDistanceRequest = TransactionUtils.isDistanceRequest(transaction);
+    const isFetchingWaypointsFromServer = TransactionUtils.isFetchingWaypointsFromServer(transaction);
+    const isPendingDistanceRequest = isDistanceRequest && isFetchingWaypointsFromServer && !requestAmount;
```

Comment:
`isDistanceRequest && isFetchingWaypointsFromServer` is redundant, isn't it? A transaction won't be fetching waypoints from the server unless it's a distance request. Again here, I would just drop this variable and replace it with `isFetchingWaypointsFromServer && !requestAmount` in both places below.